src/main/java/de/tum/in/www1/artemis/domain/notification/SingleUserNotificationFactory.java

```diff
@@ -44,4 +48,40 @@ public static SingleUserNotification createNotification(Post post, NotificationT
         }
         return notification;
     }
+
+    /**
+     * Creates an instance of PlagiarismNotification
+     *
+     * @param plagiarismComparisonId ID of the plagiarism comparison associated with this case.
+     * @param recipient              affected student
+     * @param author                 instructor creating notification
+     * @param message                instructors message
+     * @return the PlagiarismNotification
+     */
+    private static SingleUserNotification createPlagiarismNotification(Long plagiarismComparisonId, Long courseId, User recipient, User author, String message, boolean isUpdate) {
+        String title = isUpdate ? "Update on your plagiarism case" : "Possible case of Plagiarism";
+        if (recipient.getLangKey() != null) {
+            Locale locale = Locale.forLanguageTag(recipient.getLangKey());
+            if (locale.equals(Locale.GERMAN) || locale.equals(Locale.GERMANY)) {
+                title = isUpdate ? "Update zu deinem Plagiatsfall" : "Möglicher plagiatsfall";
+            }
+        }
+        SingleUserNotification notification = new SingleUserNotification();
```

Comment:
Please stick to the already existing implementation as close as possible i.e. please do not use the empty constructor in the Factory but the other one 
![image](https://user-images.githubusercontent.com/65814168/143262696-c1d0e8b8-7bdc-496f-9353-b7c2d50783fb.png)
and set the neccesary properties later (just as you currently do) but the main properties can be set by the constructor. 
I know this is a nitpick but this makes the Factories (which can be quite large/complex (e.g. look at the GroupNotificationFactory) more similar and comprehensible